public/data/config.json

```diff
@@ -1,4 +1,5 @@
 {
+  "area_search_terms": true,
```

Comment:
nit (no change needed!): one way of doing this would be to have this as a string with the URL of the file. This could make it more modular if we wanted to quickly and easily change over, as we can with the maps. (If we changed the map we would need to change this).

Reply:
I guess this would also apply to the config file. Can we put this on the issue list to cover both?

Reply:
Sounds good re putting this on backlog. I guess the way I'm thinking is that config file could have strings pointing to almost everything else.